ios/RNRudderSdkModule.m

```diff
@@ -81,6 +81,21 @@ - (dispatch_queue_t)methodQueue
     [[RSClient sharedInstance] identify:_userId traits:_traits options:[[RSOption alloc] initWithDict:_options]];
 }
 
+RCT_EXPORT_METHOD(putAndroidDeviceToken:(NSString*)token)
```

Comment:
Should be removed as per the logic specified for Android

Reply:
doen